src/sentry/south_migrations/0374_auto.py

```diff
@@ -0,0 +1,1055 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Removing M2M table for field commits on 'PullRequest'
+        db.delete_table(db.shorten_name('sentry_pull_request_commits'))
```

Comment:
it looks like you're just adding this table in the previous migration and removing here, so can you just regenerate the previous migration to not include this table?

Reply:
oops, thanks